modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/TypeUtils.java

```diff
@@ -715,16 +717,22 @@ private static TypeSpec convertToTypeSpec(RelDataType type) {
      *  <br>
      *  2) If one or more of the rightmost M-L characters of V are not space(s), then an
      *   exception condition is raised: data exception — string data, right truncation.
+     *  <br>
+     *  3) If the declared type of T is binary string and the length in octets of V is greater than the
```

Comment:
you need to restructure javadoc: I believe this item doesn't relate to `fixed-length character string`.

Besides, may you point at particular section from standard? Can't find this paragraph in Store assignment section

Reply:
good catch, really i have read not related part (( fixed